src/protagonist/API/Features/Space/SpaceController.cs

```diff
@@ -140,27 +138,31 @@ public async Task<IActionResult> DeleteSpace(int customerId, int spaceId)
     [Route("{spaceId}")]
     [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(DLCS.HydraModel.Space))]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
-    public async Task<IActionResult> GetSpace(int customerId, int spaceId)
+    public async Task<IActionResult> GetSpace(
+        [FromRoute] int customerId, 
+        [FromRoute] int spaceId,
+        CancellationToken cancellationToken)
     {
-        var dbSpace = await Mediator.Send(new GetSpace(customerId, spaceId));
-        if (dbSpace != null)
-        {
-            return Ok(dbSpace.ToHydra(GetUrlRoots().BaseUrl));
-        }
-
-        return NotFound();
+        var request = new GetSpace(customerId, spaceId);
+        
+        return await HandleFetch(
+            request,
+            s => s.ToHydra(GetUrlRoots().BaseUrl),
+            errorTitle: "Failed to get space",
+            cancellationToken: cancellationToken
+        );
     }
 
     /// <summary>
-    /// Make a partial update of an existing space
+    /// Patch a space within this customer. DLCS assigns identity.
```

Comment:
The previous xml docs were more accurate, `DLCS assigns identify` isn't relevant here as this is a partial update to an existing space so we'd be unaware how the id was assigned.

Reply:
I'm not sure why I edited it - I think it might've been by mistake while I was working on the comments for PUT. Thanks for spotting it!